Sources/Fuzzilli/FuzzIL/JsOperations.swift

```diff
@@ -1047,21 +1047,37 @@ final class TestIn: JsOperation {
 
 }
 
+enum ParameterPattern {
```

Comment:
I think it would also be good to add support to our code generation machinery to generate functions with more complex parameters. The place for that would be here: https://github.com/googleprojectzero/fuzzilli/blob/7b2850d6831535aedba74da7f8811dda6ee59abd/Sources/Fuzzilli/Base/ProgramBuilder.swift#L425 Maybe this makes most sense as a follow-up change though, so we can land this one first to get the compiler support and then see if it's easy to also get codegen support.

Reply:
Referenced this in a new issue

Reply:
Thanks!